app/src/main/java/org/simple/clinic/drugstockreminders/DrugStockReminderApi.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.drugstockreminders
+
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.GET
+
+interface DrugStockReminderApi {
+
+  @GET("v4/drug_stock")
+  fun drugStockReminder(
```

Comment:
I am guessing we are going to add API integration test later? Fwiw we can write the test and use `Ignore` annotation to skip that test until server implements it.

Reply:
Yes we're going to add it in the next PR.